### ISSUE TITLE
Remove redundant visually hidden "(current)" from pagination controls

### ISSUE THREAD

**Issue by ihorzenich** [ng-bootstrap/ng-bootstrap#3870]
TL;DR - Remove the `<span class="visually-hidden">(current)</span>` where the controls already have `aria-current`

### Bug description:
Keeping `<span class="sr-only">(current)</span>` together with `aria-current="page"` on parent block cause double announcement of "current" page in NVDA screen-reader.
![image](https://user-images.githubusercontent.com/1684521/95966239-901e3b80-0e13-11eb-8c20-89c6e7ccaad8.png)
It is necessary to remove `<span class="sr-only">(current)</span>` from `ngb-pagination`.

This has already been done in Bootstrap: https://github.com/twbs/bootstrap/pull/31892
I think ng-bootstrap pagination should also be in line with this.

### Link to minimally-working StackBlitz that reproduces the issue:
https://ng-bootstrap.github.io/stackblitzes/pagination/basic/stackblitz.html

### Versions of Angular, ng-bootstrap and Bootstrap:
Angular: 10.0.0
ng-bootstrap: 7.0.0
Bootstrap: 4.5.4

**Comment by RahulBavaliya** [comment]
You can use below css code for hidding this span tag


```
:host ::ng-deep .pagination > li.page-item.active > a.page-link > span {
    display: none;
}

```
